docs/README.md

```diff
@@ -73,20 +73,21 @@ Legend:
 | OS Flavor / version | 23.1 Enterprise/Community | 7.4 Enterprise     | 7.3 Enterprise     | 7.2 Enterprise     | 7.1 Enterprise     | 7.0 Enterprise     |
 |---------------------|---------------------------|--------------------|--------------------|--------------------|--------------------|--------------------|
 | Amazon Linux (v2)   | :x:                       | :x:                | :x:                | :x:                | :x:                | :x:                |
-| Amazon Linux (v1)   | :x:                       | :x:                | :x:                | :x:                | :x:                | :x:                |
-| RHEL 9.2            | :white_check_mark:        | :x:                | -                  | -                  | -                  | -                  |
-| RHEL 8.7            | :white_check_mark:        | :white_check_mark: | -                  | -                  | -                  | -                  |
-| RHEL 8.6            | :white_check_mark:        | :white_check_mark: | :white_check_mark: | -                  | -                  | -                  |
-| RHEL 8.5            | :white_check_mark:        | :white_check_mark: | :white_check_mark: | :white_check_mark: | -                  | -                  |
-| RHEL 8.4            | -                         | -                  | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
-| RHEL 8.2            | -                         | -                  | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| Amazon Linux (v1)   | -                         | -                  | :x:                | -                  | :x:                | :x:                |
+| RHEL 9              | :white_check_mark:        | -                  | -                  | -                  | -                  | -                  |
+| RHEL 8.8            | :white_check_mark:        | -                  | -                  | -                  | -                  | -                  |
+| RHEL 8.7            | -                         | :white_check_mark: | -                  | -                  | -                  | -                  |
+| RHEL 8.6            | -                         | :white_check_mark: | :white_check_mark: | -                  | -                  | -                  |
+| RHEL 8.5            | -                         | -                  | :white_check_mark: | :white_check_mark: | -                  | -                  |
+| RHEL 8.4            | -                         | -                  | -                  | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+| RHEL 8.2            | -                         | -                  | -                  | :white_check_mark: | :white_check_mark: | :white_check_mark: |
```

Comment:
I am a bit worried that in the end here we are mirroring the docs of the standard supported platforms, except for the OS we don't explicitly support.

Can't we just say the playbook don't support Amazon Linux and Suse and please refers to the supported platform page to find the supported os version?

Reply:
Sounds better indeed.
@anxumalo We will remove this table to avoid discrepencies as we are pretty bad at maintaining it. We'll just mention the exceptions we have compare to what's in docs. Might need an update on Ansible docs as well